tests/parametric/test_otel_span_methods.py

```diff
@@ -541,5 +563,8 @@ def run_otel_span_reserved_attributes_overrides_analytics_event(
     assert len(trace) == 1
 
     span = get_span(test_agent)
-    assert span["metrics"].get("_dd1.sr.eausr") == expected_metric_value
+    if (expected_metric_value is not None) or (context.library not in ["dotnet"]):
```

Comment:
Hmm why are we checking for library here and not using the `missing_feature` decorator?